### PR TITLE
Fix for clear orphan events work flow.

### PR DESCRIPTION
**Pull request by khandelwalbrijesh**
https://github.com/microsoft/Service-Fabric-POA/issues/43

### CODE REVIEW

File: src/PatchOrchestrationApplication/CoordinatorService/src/HealthManagerHelper.cs [hunk 1]
```diff
@@ -41,5 +41,34 @@ class HealthManagerHelper
...
+        /// Posts a health report against the current service to indicate status of Patch Orchestration Agent
+        /// </summary>
+        /// <param name="fabricClient">Fabric client object to carry out HM operations</param>
+        /// <param name="serviceUri">Uri of the service against which health report is to be posted</param>
+        /// <param name="sourceId">SourceId of the health report</param>
+        /// <param name="healthReportProperty">Property of the health report</param>
+        /// <param name="description">Description of the health report</param>
+        /// <param name="healthState">HealthState for the health report</param>
+        /// <param name="timeToLiveInMinutes">Time to live in minutes for health report</param>
+        internal static void UpdateHealthReport(FabricClient fabricClient, Uri serviceUri, string sourceId, string healthReportProperty, string description,
```
Comment (raunakpandya) [inline src/PatchOrchestrationApplication/CoordinatorService/src/HealthManagerHelper.cs]: >UpdateHealthReport [](start = 29, length = 18)

Whats the difference between this and above method?

File: src/PatchOrchestrationApplication/CoordinatorService/src/RepairManagerHelper.cs [hunk 2]
```diff
@@ -431,21 +430,16 @@ public async Task ClearOrphanEvents(CancellationToken cancellationToken)
                         propertyDict.Add(WUOperationStatus + "-" + node.NodeName, true);
                         propertyDict.Add(WUOperationSetting + "-" + node.NodeName, true);
                     }
+
+                    string NodeNotPartOfClusterDescription = "This node is no longer part of the cluster. This health event will expire in 1 minute.";
```
Comment (raunakpandya) [inline src/PatchOrchestrationApplication/CoordinatorService/src/RepairManagerHelper.cs]: >This health event will expire in 1 minute. [](start = 106, length = 42)

We can remove the part - "This health event will expire in 1 minute."

File: src/PatchOrchestrationApplication/CoordinatorService/src/HealthManagerHelper.cs (hunk 1 above)
Comment (khandelwalbrijesh) [reply src/PatchOrchestrationApplication/CoordinatorService/src/HealthManagerHelper.cs]: It updates the sourceID as well

**Comment by raunakpandya** [comment]
> 

In my opinion we should remove this source id defined here and have just one method below which always take in source id. Since this is a helper method.

---
Refers to: src/PatchOrchestrationApplication/CoordinatorService/src/HealthManagerHelper.cs:16 in 4273d80. [](commit_id = 4273d80602bcb4efa3118fa67dd0158a12a966a1, deletion_comment = False)

**Pull request merged**